core/kazoo_provisioner/src/provisioner_v5.erl

```diff
@@ -223,13 +223,21 @@ set_owner(Device) ->
 %%------------------------------------------------------------------------------
 -spec settings(kzd_devices:doc()) -> kz_json:object().
 settings(Device) ->
+    settings(Device, kz_doc:type(Device)).
+
+-spec settings(kzd_devices:doc(), kz_term:ne_binary()) -> kz_json:object().
+settings(Device, <<"device">>) ->
     Props = props:filter_empty([{<<"lines">>, settings_lines(Device)}
                                ,{<<"codecs">>, settings_codecs(Device)}
                                ,{<<"datetime">>, settings_datetime(Device)}
                                ,{<<"feature_keys">>, settings_feature_keys(Device)}
                                ,{<<"line_keys">>, settings_line_keys(Device)}
                                ,{<<"combo_keys">>, settings_combo_keys(Device)}
                                ]),
+    kz_json:from_list(Props);
+settings(Device, _) ->
```

Comment:
Obviously not a device JObj right? :) and update the type from `kzd_devices:doc()` to `kz_json:object()`.